pinc/POFile.inc

```diff
@@ -126,76 +128,68 @@ class POFile
 
     public function create_template($basedir, $pot_filename)
     {
-        $cmd = join(" ", [
-            // change into the base directory so the filenames in the POT
-            // don't have it prefixed to them
-            "cd " . escapeshellarg($basedir),
-            "&&",
+        // run from the base directory so the filenames in the POT
+        // don't have it prefixed to them
+        $process = new Process([
             "./vendor/bin/xtext",
-            "-x=./vendor",
-            "-x=./SETUP",
-            "-i='*.php'",
-            "-i='*.inc'",
-            "-c=TRANSLATORS",
+            "-x", "./vendor",
+            "-x", "./SETUP",
+            "-i", "*.php",
+            "-i", "*.inc",
+            "-c", "TRANSLATORS",
             ".",
-            "2>&1",
-            ">",
-            escapeshellarg($pot_filename),
-        ]);
-        exec($cmd, $exec_out, $return_value);
-        if ($return_value != 0) {
-            throw new Exception(implode("\n", $exec_out));
+            $pot_filename,
```

Comment:
Does xtext use this as an output file name, so you don't have to use output redirection?

Reply:
Oh, no, I messed this one up. Thanks for catching that!

Reply:
Fixed this and made the new logic nicer in case `xtext` fails. Requires jumping through a few more hoops but its worth it.